### PR TITLE
Improve clarity on the description of the "Min number of characters fo…

### PR DESCRIPTION
**Pull request by ooker777**
…r trigger" setting

Sorry, I would not accept the pull requests except for the following cases.

Fix obvious bugs
Fix typo or wrong documentation
If I ask for it in the GitHub issues or the discussions

### CODE REVIEW

File: src/setting/settings.ts [hunk 1]
```diff
@@ -417,7 +417,7 @@ export class VariousComplementsSettingTab extends PluginSettingTab {
 
     new Setting(containerEl)
       .setName("Min number of characters for trigger")
-      .setDesc("It uses a default value of Strategy if set 0.")
+      .setDesc("Setting the value to 0 does not mean the suggestion will be triggered without any inputted character. Depending on the Strategy you choose the actual value will be differ. Read the docs for more info.")
```
Comment (tadashi-aikawa) [inline src/setting/settings.ts:420]: @ooker777 
Thank you. It became very clear! What do you think of making a minor adjustment to the following?

---

Setting the value to 0 does not mean that suggestions will be triggered without any inputted characters. Depending on the "Strategy" you choose, the preset value will be used.

---

I will add a link to the documentation for "the preset value".

Comment (ooker777) [reply src/setting/settings.ts:420]: Yes, "preset" is a better word. However can you modify the preset value? The word seems to imply that you can change it to your preference

Comment (tadashi-aikawa) [reply src/setting/settings.ts:420]: > However can you modify the preset value? The word seems to imply that you can change it to your preference

No, we can't change the value. 

How about the following?

```diff
- Depending on the "Strategy" you choose, the preset value will be used.
+ Depending on the "Strategy" you choose, a designated value will be used for each.
```

Comment (ooker777) [reply src/setting/settings.ts:420]: Yes, I think that will work

Comment (tadashi-aikawa) [reply src/setting/settings.ts:420]: @ooker777 
OK. Please fix it and commit 🙏

**Pull request merged**